Documentation/MARKETPLACE.md

```diff
@@ -58,7 +58,7 @@ First, you must work with the CSSA team to get an Azure subscription setup for y
 10. The last important tab is the "Tags". The application will automatically generate a few tags and apply them to your installation. Youo can use this section to add your specific tags (name/value pairs).
```

Comment:
Hi William, thanks for your contribution.  Can you also fix this "Youo" we missed?

Reply:
Added in commit `f03b34c`